### ISSUE TITLE
show string property values w/o quotes

### ISSUE THREAD

**Issue by devoncarew** [flutter/flutter-intellij#4210]
For string valued properties, we may choose to show the string value without quotes - users would likely expect to be able to type into the text field directly w/o also having to have starting and ending quotes.

<img width="306" alt="Screen Shot 2019-12-12 at 2 13 01 PM" src="https://user-images.githubusercontent.com/1269969/70753504-c073e580-1ce9-11ea-9af4-b3146d0f2ecd.png">

<img width="283" alt="Screen Shot 2019-12-12 at 2 13 10 PM" src="https://user-images.githubusercontent.com/1269969/70753513-c4076c80-1ce9-11ea-9985-07471e158e8b.png">

cc @jacob314

**Comment by jacob314** [comment]
I agree this is good polish. The one question is how should we disambiguate between
```dart
  title: planetNames
```
and
```dart
  title: 'planetNames'
```

Options:
1. we could show existing expressions as read only and only allow editing if what is specified is purely a string literal.

2. We could provide some visual indication in the text field showing whether it is a string literal of a Dart expression. Maybe some UI like an icon indicating `expression` on the left side of the checkbox would work. You could click on the icon to toggle the current editing mode between string literal and expression.

@InMatrix @devoncarew any thoughts?
It is nice to be able to show and tweak expressions but I agree that support gets in the way of making string literals as easy as they could be to specify currently.

**Comment by devoncarew** [comment]
I like option 1 - only allow editing if the property value is a literal. For cases of expressions, we might have an easy way to jump from the property value to the corresponding source code?

**Comment by pq** [comment]
This request for the legacy Flutter Outline View. Closing as stale.
